x-pack/plugins/actions/server/builtin_action_types/cases_webhook/validators.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as i18n from './translations';
+import { ActionsConfigurationUtilities } from '../../actions_config';
+import {
+  CasesWebhookPublicConfigurationType,
+  CasesWebhookSecretConfigurationType,
+  ExternalServiceValidation,
+} from './types';
+
+const validateConfig = (
+  configurationUtilities: ActionsConfigurationUtilities,
+  configObject: CasesWebhookPublicConfigurationType
+) => {
+  const {
+    createCommentUrl,
+    createIncidentUrl,
+    incidentViewUrl,
+    getIncidentUrl,
+    updateIncidentUrl,
+  } = configObject;
```

Comment:
If we change these types in `x-pack/plugins/actions/server/builtin_action_types/cases_webhook/schema.ts`
from  ```schema.string()``` 
to ```schema.uri({ scheme: ['http', 'https'] })```

example: packages/kbn-config-schema/src/types/uri_type.test.ts

Would it be able to skip the check of ```new URL(url)``` on Line 39 in here?

Reply:
ah just found that schema.url doesn't accept {{{}}} in an url, it always regards it as an invalid url

Reply:
i took care of this on the service function level for each compiled mustache url